homepage.md

```diff
@@ -32,6 +32,9 @@ Statistics about stored and transferred data:
 - **storage_total_pieces** - total number of pieces stored on storage nodes
 - **storage_total_segments** - total number of segments stored on storage nodes
 - **storage_free_capacity_estimate_bytes** - statistical estimate of free storage node capacity, with suspicious values removed
+- **storage_after_expansion_total_bytes** - total number of bytes stored on the network after expansion
+- **vetted_median_available_bytes** - vetted nodes median available bytes
```

Comment:
I am not sure what this means.

Reply:
I think the problem here is the nomenclature.
 **storage_after_expansion_total_bytes** represents the total amount of storage used by all Nodes after the data expansion process using Reed Solomon. On the other hand, **storage_total_bytes** refers to the original amount of data uploaded before expansion.

 Example:

    Total uploaded 3Tb (before expansion)
    Total stored 5Tb (after expansion)

    In this case we have a expansion factor of ~ 1.67

There is another metric called **median_available_bytes**, which shows the median amount of bytes stored by all nodes (including unvetted ones). This metric was previously undocumented, but I will update the documentation with a commit.

Example:

```
    Node A: 10Gb
    Node B: 5Gb
    Node C: 12Gb
    Node D: 23Gb

    Median: 11Gb
```

**vetted_median_available_bytes**  shows the median available bytes considering only vetted nodes.

Reply:
Thanks for clarifying! However, it seams the "median" metrics are wrong because currently they show 11PB or 12PB per node, which is too large.

I suggest the following names based on the already established naming convention:

- **storage_total_bytes_after_expansion** - total number of bytes stored on the network after expansion by erasure coding
- **storage_free_capacity_estimate_vetted_bytes** - statistical estimate of free storage node capacity considering only vetted nodes
- **storage_free_capacity_median_bytes_per_node** - median free capacity per storage node (unvetted nodes included)
- **storage_free_capacity_median_bytes_per_vetted_node** - median free capacity per storage node (vetted nodes only)

Please insert them lexicographically sorted into the existing list.

Reply:
After reviewing the metrics, I found that the median is not displaying the information I previously mentioned. Instead, it shows the sum of available space of nodes that belong to the 50th percentile (median). We can postpone all median related metrics since it will not add much information and may cause confusion. What do you think?

Reply:
Why did we add these median metrics? I guess someone needed that info. But if not, I am fine with removing them for now.

Reply:
We currently estimate free storage using the 95th percentile. However, I also have calculations for the 50th and 75th percentiles. I thougt the 50th percentile (median) could be useful for someone 💁 . Although I didn't create an entry on `homepage.md` for this information. I'm going to push another commit lexicographically sorted and removing the median entries